src/cordova/plugin/apple/scripts/copy_third_party_frameworks_ios.js

```diff
@@ -18,10 +18,16 @@ const child_process = require('child_process');
 
 module.exports = function(context) {
   console.log('Copying iOS third party frameworks');
+  child_process.execSync('rm -rf plugins/cordova-plugin-outline/apple/lib/ios');
```

Comment:
*nit* does it make more sense to `rm` in `clean` phase?

Reply:
It fails to copy otherwise.